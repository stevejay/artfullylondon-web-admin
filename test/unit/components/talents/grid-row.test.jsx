import React from 'react'
import { shallow } from 'enzyme'
import _ from 'lodash'

import TalentsGridRow from '_src/components/talents/grid-row'

it('should render correctly', () => {
  const wrapper = shallow(
    <TalentsGridRow
      value={{
        key: 'some-key',
        id: 'some-id',
        talentType: 'Actor',
        roles: 'The Roles',
        characters: 'The Characters',
        firstNames: 'First Names',
        lastName: 'LastName',
        name: 'The Name'
      }}
      onDelete={_.noop}
      onRolesChange={_.noop}
      onCharactersChange={_.noop}
    />
  )

  expect(wrapper).toMatchSnapshot()
})
