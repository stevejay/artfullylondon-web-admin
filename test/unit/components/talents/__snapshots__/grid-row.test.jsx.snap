// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<GridRow
  id="some-key"
  onDelete={[Function]}
>
  <div
    styleName="first-row"
  >
    <a
      href="/talent/some-id"
      rel="noopener"
      target="_blank"
    >
      The Name
    </a>
  </div>
  <Text
    autos={true}
    disabled={false}
    forceSingleLine={true}
    maxLength={100}
    onChange={[Function]}
    password={false}
    placeholder="Roles"
    readOnly={false}
    style={
      Object {
        "marginBottom": ".5rem",
      }
    }
    value="The Roles"
  />
  <Text
    autos={true}
    disabled={false}
    forceSingleLine={true}
    maxLength={100}
    onChange={[Function]}
    password={false}
    placeholder="Characters"
    readOnly={false}
    value="The Characters"
  />
</GridRow>
`;
